apps/web/app/actions/doers.ts

```diff
@@ -265,6 +262,40 @@ export const createMemory = async (input: {
 
   let contentId: number | undefined;
 
+  const response = (await vectorSaveResponse.json()) as {
+    status: string;
+    chunkedInput: string;
+    message?: string;
+  };
+
+  try {
+    if (response.status !== "ok") {
+      if (
+        response.status === "error" &&
+        response.message === "no chunks over 20"
```

Comment:
matching the exact message can make it difficult to maintain, we can either:
1. use `data` as an numeric error code
2. prefix the message with 'ERROR:' and show the exact same error message to the user

Reply:
Yeah, gonna change this